x/subscription/client/cli/tx_buy.go

```diff
@@ -11,31 +9,35 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var _ = strconv.Itoa(0)
-
-func CmdSubscribe() *cobra.Command {
+func CmdBuy() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "subscribe [consumer] [index] [is-yearly]",
```

Comment:
if params are optional, explain it in the use

Reply:
ok.